dnsmasq/config.json

```diff
@@ -30,6 +32,20 @@
         "host": "str",
         "ip": "str"
       }
+    ],
+    "services": [
+      {
+        "srv": "str",
+        "host": "str",
+        "port": "str",
+        "priority": "int",
+        "weight": "int"
+      }
+    ],
+    "extra_options": [
```

Comment:
I don't like common options that get not validate and end up in a not working state. Can you not expose on/off function for that options which you need?